core/src/services/hdfs/docs.md

```diff
@@ -121,6 +121,7 @@ async fn main() -> Result<()> {
     // Create fs backend builder.
     let mut builder = Hdfs::default();
     // Set the name node for hdfs.
+    // If the string starts with a protocol type such as file://, hdfs://, or gs://, this protocol type will be used.
```

Comment:
Note: This note comes from https://docs.rs/hdrs/latest/hdrs/struct.ClientBuilder.html#method.new.

If using gcs, we simply need to pass in `gs://${bucket_name}/` as the name node, and keep `root` unchanged. All data access operations will be under `gs://${bucket_name}${root}`